platform/NET/build.mk

```diff
@@ -5,3 +5,6 @@
 
 PLATFORM_NET_PATH = $(PLATFORM_MODULE_PATH)/NET/$(PLATFORM_NET)
 include $(call rwildcard,$(PLATFORM_NET_PATH)/,sources.mk)
+
+# Stick to some POSIX-conforming API to disable BSD extensions
+CFLAGS += -D_POSIX_C_SOURCE=200809
```

Comment:
`time.h` started to include `select.h` (indirectly) which contains declarations conflicting with CC3000. Thankfully, it's guarded by `__BSD_VISIBLE` which is not defined when POSIX-compliant API is requested explicitly.